modules/apps/ai-creator/ai-creator-openai-web/src/main/java/com/liferay/ai/creator/openai/web/internal/portlet/action/UploadGenerationsMVCActionCommand.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.ai.creator.openai.web.internal.portlet.action;
+
+import com.liferay.ai.creator.openai.web.internal.constants.AICreatorOpenAIPortletKeys;
+import com.liferay.ai.creator.openai.web.internal.upload.AICreatorOpenAIUploadFileEntryHandler;
+import com.liferay.document.library.kernel.service.DLAppService;
+import com.liferay.portal.kernel.portlet.bridges.mvc.BaseMVCActionCommand;
+import com.liferay.portal.kernel.portlet.bridges.mvc.MVCActionCommand;
+import com.liferay.portal.kernel.util.File;
+import com.liferay.upload.UploadHandler;
+import com.liferay.upload.UploadResponseHandler;
+
+import java.util.Map;
+
+import javax.portlet.ActionRequest;
+import javax.portlet.ActionResponse;
+
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Modified;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Roberto Díaz
+ */
+@Component(
+	property = {
+		"javax.portlet.name=" + AICreatorOpenAIPortletKeys.AI_CREATOR_OPENAI,
+		"mvc.command.name=/ai_creator_openai/upload_generations"
+	},
+	service = MVCActionCommand.class
+)
+public class UploadGenerationsMVCActionCommand extends BaseMVCActionCommand {
```

Comment:
What do you mean by "generations" here? Shouldn't it be "generated images"?

Reply:
It's how we are calling the image generations:

com.liferay.ai.creator.openai.web.internal.client.AICreatorOpenAIClient#getCompletion#getCompletion -> com.liferay.ai.creator.openai.web.internal.portlet.action.GetCompletionMVCResourceCommand


com.liferay.ai.creator.openai.web.internal.client.AICreatorOpenAIClient#getGenerations -> 
com.liferay.ai.creator.openai.web.internal.portlet.action.GetGenerationsMVCResourceCommand

Son in this case: UploadGenerationsMVCActionCommand follows this pattern